autogen/retrieve_utils.py

```diff
@@ -94,7 +94,11 @@ def split_text_to_chunks(
                 f"max_tokens is too small to fit a single line of text. Breaking this line:\n\t{lines[0][:100]} ..."
             )
             if not must_break_at_empty_line:
-                split_len = int(max_tokens / lines_tokens[0] * 0.9 * len(lines[0]))
+                denominator = lines_tokens[0] * 0.9 * len(lines[0])
```

Comment:
Could you please add  some tests to cover the added code?

Reply:
```
split_len = int(max_tokens / (lines_tokens[0] * 0.9 * len(lines[0]) + 0.1))
```
This would be enough. When a chunk_size is too small to fit a single line of text, why would the text have 0 tokens or 0 lenght?